IEX.h

```diff
@@ -64,18 +64,17 @@ namespace IEX {
             curl_easy_setopt(curl, CURLOPT_FOLLOWLOCATION, 1L);
             long int httpCode(0);
 
-            std::stringstream httpData;
+            std::unique_ptr<std::string> httpData(new std::string);
 
             curl_easy_setopt(curl, CURLOPT_WRITEFUNCTION, callback);
             curl_easy_setopt(curl, CURLOPT_WRITEDATA, httpData.get());
             curl_easy_perform(curl);
             curl_easy_getinfo(curl, CURLINFO_RESPONSE_CODE, &httpCode);
             curl_easy_cleanup(curl);
 
-            Json::CharReaderBuilder jsonReader;
-            std::string errs;
-
-            Json::parseFromStream(jsonReader, httpData, &jsonData, &errs);
+            std::stringstream ss;
+            ss.str(*httpData);
+            ss >> jsonData;
```

Comment:
@rodolfobandeira this is the magic -- JsonCpp has implemented the extraction operator to pull from a stream directly into a Json::Value object, so no need for magical builder and stream parsing logic.

Reply:
@mpherg perfect shot! I just compiled here and both versions 1.7* and 1.8* works. You're killing! I'm gonna merge it and take a look on your other PRs. Thank you very much!